buildenv/jenkins/jobs/infrastructure/omrMirror

```diff
@@ -61,25 +56,26 @@ pipeline {
                                     script: 'git rev-parse --short=8 HEAD',
```

Comment:
This is unnecessary, the SHA is available in the `LAST_COMMIT` variable above.

Reply:
fixed